Automattic/wp-calypso
# Store|Woo: Automatically redirect new users to the Core UI

This PR makes the following changes during store deprecation and removal:
- After completing WooCommerce plugin installation, user will be redirected to the first step of store profiler in WooCommerce. Subsequent clicks on WooCommerce menu item will directly link to WooCommerce.
- Before installing WooCommerce plugin, WooCommerce menu item will go to Store UI dashboard for installation.

Fixes https://github.com/Automattic/wp-calypso/issues/48423

#### Screencast
https://user-images.githubusercontent.com/3747241/104184157-fab03900-544d-11eb-87ce-cf2e3dad730e.mp4

#### Testing instructions
* Start calypso with `woocommerce/store-deprecated` or `woocommerce/store-removed` enabled
* With a fresh Business Plan site, install a plugin (anything except WooCommerce) to convert the site to Atomic
* Visit the site's homepage
* Click on WooCommerce menu item
* Click "Set up my store!" button, wait until loading finishes
* Notice automatic redirection to WooCommerce first step of store profiler
